packages/liveblocks-react-ui/src/primitives/AiMessage/index.tsx

```diff
@@ -38,21 +31,6 @@ const defaultMessageContentComponents: AiMessageContentComponents = {
 /* -------------------------------------------------------------------------------------------------
  * ToolInvocationPart
  * -----------------------------------------------------------------------------------------------*/
-const AiToolInvocationContext = createContext<AiToolInvocationProps | null>(
-  null
-);
-
-export function useAiToolInvocationContext() {
-  const context = useContext(AiToolInvocationContext);
-
-  if (context === null) {
-    throw new Error(
-      "This component must be used within a tool's render method."
-    );
-  }
-
-  return context;
-}
```

Comment:
Similar to other primitives, because of `export *` being applied to this file, we can't export anything else than the components we want on `AiMessage.*`, so I moved this to `./contexts.ts`.